src/components/Amalgamation/AmalgamatingBusinesses.vue

```diff
@@ -176,6 +176,11 @@ export default class AmalgamatingBusinesses extends Mixins(CommonMixin) {
 
   readonly BusinessLookupServices = BusinessLookupServices
 
+  // If continuing a draft, initialize the amalgamatingBusinesses array as the previously saved one.
+  mounted (): void {
+    if (this.getAmalgamatingBusinesses) this.amalgamatingBusinesses = this.getAmalgamatingBusinesses
+  }
```

Comment:
OK, so you have the array in the store and a local copy here... I may rework this after you merge and I rebase.